framework/helpers/BaseStringHelper.php

```diff
@@ -338,4 +338,35 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * Masks a token to make it uncompressible.
+     * This does not secure the token since the encryption key is prepended to the result.
+     * @param string $token An unmasked token.
+     * @return string A masked token.
+     * @since 2.0.13
+     */
+    public static function maskToken($token)
```

Comment:
I prefer these methods in a SecurityHelper. Perhabs its about 'strings' but I don't see someone using these methods other than in a Security context (the component Yii::$app->security already hints to this)